### ISSUE TITLE
voidvault crash when trying to apply changes in sysctl.conf

### ISSUE THREAD

**Issue by mobinmob** [atweiden/voidvault#4]
voidvault version: 1.5.2
voidlinux iso: void-live-x86_64-20181111-xfce (official)
Installation in vbox vm, with 2gb ram, 12gb hdd

voidvault crashes with the following message:
```
* Applying /etc/sysctl.conf ...
MoarVM panic: MVM_platform_alloc_pages failed: 12
The spawned command 'void-chroot' exited unsuccessfully (exit code: 1)
  in method configure-sysctl at /home/anon/voidvault-1.5.2/lib/Voidvault/Bootstrap.pm6 
(Voidvault::Bootstrap) line 1281
  in method bootstrap at /home/anon/voidvault-1.5.2/lib/Voidvault/Bootstrap.pm6 
(Voidvault::Bootstrap) line 46
  in method new at /home/anon/voidvault-1.5.2/lib/Voidvault.pm6 (Voidvault) line 46
  in sub MAIN at /home/anon/voidvault-1.5.2/bin/voidvault line 128
  in block <unit> at /home/anon/voidvault-1.5.2/bin/voidvault line 259
```

**Comment by mobinmob** [comment]
The same crash happens with the latest official mate iso, in efi and bios mode.

**Comment by atweiden** [comment]
I suspect this is an issue with MoarVM 2018.12.

Can you re-attempt with the ISO at https://github.com/atweiden/voidiso/releases ? It ships with an earlier version of rakudo.

**Comment by mobinmob** [comment]
Using the iso from voidiso, voidvault finishes successfully :)
On reboot, there is a problem with grub, it cannot find the device to boot:
https://imgur.com/B4CpHaw
That happens on efi and bios mode.

**Comment by atweiden** [comment]
instead of running `voidvault new`, can you do `./examples/new-from-cli-passhash.sh` ? Note this will destroy your `/dev/sda`.

**Comment by atweiden** [comment]
I'm getting a strange, new error during `cryptsetup luksFormat` which I think is the source of this error.

```
Enter passphrase for /dev/sda3:
Verify passphrase:
WARNING: Locking directory /run/cryptsetup is missing!
spawn cryptsetup luksOpen /dev/sda3 vault
Enter passphrase for /dev/sda3:
btrfs-progs v4.20.1
See http://btrfs.wiki.kernel.org for more information

probe of /dev/mapper/vault failed, cannot detect existing filesystem
ERROR: use the -F option to force overwrite of /dev/mapper/vault
The spawned command `mkfs.btrfs` exited unsuccessfully (exit code: 1)
```

Have not seen this prior to `cryptsetup-2.1.0`. The same sequence of events plays out with NILFS2 and Btrfs.

The way to pin this on `cryptsetup-2.1.0` is to build `cryptsetup-2.0.6`, upload the xbps hostdir/binpkgs dir containing the built 2.0.6 package, then point voidvault to it with `voidvault --repository=/path/to/cryptosetup-2.0.6/repo`.

**Comment by mobinmob** [comment]
> I'm getting a strange, new error during `cryptsetup luksFormat` which I think is the source of this error.
> 
> ```
> Enter passphrase for /dev/sda3:
> Verify passphrase:
> WARNING: Locking directory /run/cryptsetup is missing!
> spawn cryptsetup luksOpen /dev/sda3 vault
> Enter passphrase for /dev/sda3:
> btrfs-progs v4.20.1
> See http://btrfs.wiki.kernel.org for more information
> 
> probe of /dev/mapper/vault failed, cannot detect existing filesystem
> ERROR: use the -F option to force overwrite of /dev/mapper/vault
> The spawned command `mkfs.btrfs` exited unsuccessfully (exit code: 1)
> ```
> 
> Have not seen this prior to `cryptsetup-2.1.0`. The same sequence of events plays out with NILFS2 and Btrfs.

I have the same error running ./examples/new-from-cli-passhash.sh...
I did not encounter this 13 hours ago when I tried voidlinux new. :(

**Comment by atweiden** [comment]
i can confirm this error is fixed by downgrading to cryptsetup-2.0.6

**Comment by atweiden** [comment]
fixed in latest master
